### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by IreneS1**
- how to export and require from module to module
- sync and async examples
- http server call

### CODE REVIEW

File: 01-node-tutorial/answers/prompter.js
Comment (mnca16) [inline 01-node-tutorial/answers/prompter.js:1]: Guessing game looks super organize! Great job💪

File: 01-node-tutorial/answers/prompter.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ let item = "Enter something below.";
 const form = () => {
   return `
   <body>
+  <label>Guess a number between 0 - 100</label>
   <p>${item}</p>
   <form method="POST">
   <input name="item"></input>
```
Comment (mnca16) [inline 01-node-tutorial/answers/prompter.js:34]: Will the user be able enter characters as well as number?

Comment (mnca16) [inline 01-node-tutorial/answers/prompter.js:34]: Will the user be able to enter characters as well as number?

File: 01-node-tutorial/answers/week-1/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+process.env.MY_VAR='welcome to node'
```
Comment (mnca16) [inline 01-node-tutorial/answers/week-1/.env:1]: Out of curiosity, where you able to get this value? I think we don't use "process.env." inside of the env file but I'm not sure

File: 01-node-tutorial/answers/week-1/fs-async.js
Comment (mnca16) [inline 01-node-tutorial/answers/week-1/fs-async.js:1]: Looking good!

File: 01-node-tutorial/answers/week-1/os.js
Comment (mnca16) [inline 01-node-tutorial/answers/week-1/os.js:1]: This is really cool! You took the time to dig in more into this object

**Review by mnca16 — APPROVED** [review]
Awesome work!! Everything looks super clean 😁

File: 01-node-tutorial/answers/prompter.js (hunk 1 above)
Comment (IreneS1) [reply 01-node-tutorial/answers/prompter.js:34]: I didn't add verification as I should have, so yes they could.

File: 01-node-tutorial/answers/week-1/.env (hunk 2 above)
Comment (IreneS1) [reply 01-node-tutorial/answers/week-1/.env:1]: No it didn't work. I forgot to delete this file. Ooops

**Pull request merged**